### PR TITLE
Build locally corrections

### PR DESCRIPTION
**Pull request by jadecarino**
- Updated the source maven value to match the remote builds
- Corrected typo 'ras' from 'raw'

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: build-locally.sh [hunk 1]
```diff
@@ -174,7 +174,7 @@ fi
 
 h2 "Removing .m2 artifacts"
 rm -fr ~/.m2/repository/dev/galasa/dev.galasa.cps.etcd
-rm -fr ~/.m2/repository/dev/galasa/dev.galasa.raw.couchdb
+rm -fr ~/.m2/repository/dev/galasa/dev.galasa.ras.couchdb
```
Comment (techcobweb) [inline build-locally.sh:177]: lol. A typo.

Comment (techcobweb) [reply build-locally.sh:177]: oops

**Review by techcobweb — APPROVED** [review]
All looks great. Thanks.

**Pull request merged**